Lib/test/test_buffer.py

```diff
@@ -761,10 +761,10 @@ def setUp(self):
         # The suboffsets tests need sizeof(void *).
         self.sizeof_void_p = get_sizeof_void_p()
 
-    def verify(self, result, obj=-1,
-                     itemsize={1}, fmt=-1, readonly={1},
-                     ndim={1}, shape=-1, strides=-1,
-                     lst=-1, sliced=False, cast=False):
+    def verify(self, result, *, obj,
+                     itemsize, fmt, readonly,
+                     ndim, shape, strides,
+                     lst, sliced=False, cast=False):
         # Verify buffer contents against expected values. Default values
```

Comment:
The comment that gives the rationale for the curious initialization no longer applies.

Reply:
Done in #8481.